src/po/sr.po

```diff
@@ -358,190 +623,144 @@ msgstr "E787: Бафер је неочекивано измењен"
+msgid "E791: Empty keymap entry"
+msgstr "E791: Празна keymap ставка"
 
 msgid "E111: Missing ']'"
 msgstr "E111: Недостаје ']'"
 
 msgid "E719: Cannot use [:] with a Dictionary"
 msgstr "E719: Не може да се користи [:] са Речником"
 
-#, c-format
-msgid "E734: Wrong variable type for %s="
-msgstr "E734: Погрешан тип променљиве за %s="
-
-#, c-format
-msgid "E461: Illegal variable name: %s"
-msgstr "E461: Недозвољено име променљиве: %s"
-
 msgid "E806: using Float as a String"
 msgstr "E806: коришћење Float као String"
 
-msgid "E687: Less targets than List items"
-msgstr "E687: Мање одредишта него ставки Листе"
-
-msgid "E688: More targets than List items"
-msgstr "E688: Више одредишта него ставки Листе"
-
-msgid "Double ; in list of variables"
-msgstr "Дупле ; у листи променљивих"
-
-#, c-format
-msgid "E738: Can't list variables for %s"
-msgstr "E738: Не може да се прикаже листа променљивих за %s"
+msgid "E274: No white space allowed before parenthesis"
+msgstr "E274: Испред заграде није дозвољен празан карактер"
 
-msgid "E689: Can only index a List or Dictionary"
-msgstr "E689: Само Листа или Речник могу да се индексирају"
+msgid "E689: Can only index a List, Dictionary or Blob"
+msgstr "E689: Само Листа, Речник или Blob могу да се индексирају"
 
 msgid "E708: [:] must come last"
 msgstr "E708: [:] мора да буде последња"
 
-msgid "E709: [:] requires a List value"
-msgstr "E709: [:] захтева вредност типа List"
+msgid "E709: [:] requires a List or Blob value"
+msgstr "E709: [:] захтева вредност типа List или Blob"
+
+msgid "E972: Blob value does not have the right number of bytes"
+msgstr "E972: Blob вредност нема одговарајући број бајтова"
+
+msgid "E996: Cannot lock a range"
+msgstr "E996: Опсег не може да се закључа"
 
 msgid "E710: List value has more items than target"
 msgstr "E710: Вредност типа List има више ставки него одредиште"
 
 msgid "E711: List value has not enough items"
 msgstr "E711: Вредност типа List нема довољно ставки"
 
+msgid "E996: Cannot lock a list or dict"
+msgstr "E996: Листа или Речник не могу да се закључају"
+
 msgid "E690: Missing \"in\" after :for"
 msgstr "E690: Недостаје \"in\" након :for"
 
-#, c-format
-msgid "E108: No such variable: \"%s\""
-msgstr "E108: Не постоји таква променљива: \"%s\""
-
-#, c-format
-msgid "E940: Cannot lock or unlock variable %s"
-msgstr "E940: Не може да се откључа или закључа променљива %s"
-
-msgid "E743: variable nested too deep for (un)lock"
-msgstr "E743: променљива је угњеждена сувише дубоко да би се за(от)кључала"
-
 msgid "E109: Missing ':' after '?'"
 msgstr "E109: Недостаје ':' након '?'"
 
 msgid "E804: Cannot use '%' with Float"
 msgstr "E804: '%' не може да се користи са Float"
 
+msgid "E973: Blob literal should have an even number of hex characters"
+msgstr "E973: Blob литерал би требало да има паран број хекс карактера"
+
 msgid "E110: Missing ')'"
 msgstr "E110: Недостаје ')'"
 
+msgid "E260: Missing name after ->"
+msgstr "E260: Недостаје име након ->"
```

Comment:
Maybe "Ime nedostaje nakon ->"? I think it reads a bit better. Same below for the mentions of "Blob".